modules/persistent-storage-csi-vsphere-encryption-tag-based.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-encryption-tag-based_{context}"]
+= Using tag-based placement
+
+.Procedure
+
+To encrypt using tag-based placement:
+
+. In vCenter create a category for tagging datastores that will be made available to this storage class. Also, ensure that StoragePod, Datastore, and Folder are selected as Associable Entities for the created category.
+
+. In vCenter, create a tag that uses the category created earlier.
```

Comment:
If you add a step to navigate to vCenter, you can remove the "In vCenter" from any relevant steps. The "...created earlier" and "previously created" (in the next step) phrasing starts to muddle the meaning a bit, so I would suggest something like this: `. Create a tag that uses the new category.`